src/app/demo/form/form-test/form-test.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-form-test',
+  templateUrl: './form-test.component.html',
+  styleUrls: ['./form-test.component.scss']
+})
+export class FormTestComponent {
```

Comment:
Der Name ist zu allgemein. Besser wäre z.B. WebFontTestComponent, WebFontDemoComponent,...

Reply:
Der Name wurde in  WebFontDemoComponent geändert.